README.md

```diff
@@ -44,47 +44,87 @@ module.exports = {
 
 ## Definitions
 
-TODO describe definition
+Definitions are required for the raincatcher-angularjs portal application to define a workflow step. Definitions are defined the `definitions.json` file which is directly linked to the angular directives names through the templates field. The name and description fields will be the step name and decription in the portal·
+
 ```json
-{
-  "code": "base-form",
-  "name": "Base Form",
-  "description": "base form used to for steps",
-  "templates": {
-    "form": "<base-form></base-form>",
-    "view": "<base></base>"
-  }
-}
+[
+    {
+        "code": "base-form",
+        "name": "Base Form",
+        "description": "base form used to for steps",
+        "templates": {
+            "form": "<base-form></base-form>",
+            "view": "<base></base>"
+        }
+    }, 
+    ...
+]
 ```
 
 
 ## Naming conventions
 
-What naming converstions we use
+What naming conventions 
+- Angular directive uses camel case e.g. `baseForm` 
+- File name and `definition.json` use hifen e.g. 
+
+file: 
+    
+    base-form.tpl.html
+
+definition.json:
+```json
+"templates": {
+    "form": "<base-form></base-form>",
```

Comment:
Another invalid json? Why we removed rest of the content. If you want to get more lenient parsing use javascript type.